tests/lit_gradio_interface_demo.py

```diff
@@ -0,0 +1,57 @@
+import gradio as gr
```

Comment:
[example](https://github.com/Lightning-AI/lightning/tree/master/examples) folder would be a good place for this.

Reply:
`examples/app/gradio_custom_css/app.py`

Reply:
also would be a good idea to add the CSS file